docs/widgets.md

```diff
@@ -14,28 +14,28 @@ The grid sprite is a 3x3 grid and with 9 blocks, representing the 4 corners, the
 
 The corners are drawn at the same size, the sides are stretched on the side direction and the middle is expanded both ways.
 
-The `NineTileBox` widget implements a Container using that standard. This pattern is also implemented in game by the `NineTileBoxComponent` where you can use this same feature, but directly into the game Canvas, to know more about this, check the component docs [here](doc/components.md#nine-tile-box-component).
+The `NineTileBox` widget implements a Container using that standard. This pattern is also implemented in game by the `NineTileBoxComponent` where you can use this same feature, but directly into the game Canvas, to know more about this, check the component docs [here](/docs/components.md#nine-tile-box-component).
 
 Here you can find an example of its usage:
 
 ```dart
 import 'package:flame/widgets/nine_tile_box.dart';
 
 NineTileBox(
-        image: image, // dart:ui image instance
-        tileSize: 16, // The width/height of the tile on your grid image
-        destTileSize: 50, // The dimensions to be used when drawing the tile on the canvas
+        image: image,        // dart:ui image instance
+        tileSize: 16,        // The width/height of the tile on your grid image
+        destTileSize: 50,    // The dimensions to be used when drawing the tile on the canvas
```

Comment:
I actually prefer the comments close to the code, I think it looks odd with spaces extra spacing

Reply:
Fixed!